server/static/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
Why has this file been added?

Reply:
I have removed this, I have unnecessarily added it while trying to work out some errors I was getting while trying to figure out the template